presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -133,6 +133,12 @@ public void testCreateTable()
         assertTrue(getQueryRunner().tableExists(getSession(), "test_create"));
         assertTableColumnNames("test_create", "a", "b", "c");
 
+        assertQueryFails("CREATE TABLE test_create (a bigint, b double, c varchar)",
+                "(.*)Table '(\\w+).tpch.test_create' already exists");
```

Comment:
I think this is easier to read without wrapping, since it keeps the SQL together